doc/obs_integration.md

```diff
@@ -10,7 +10,9 @@
     - [GitHub Fork](#github-fork)
     - [Configuring the GitHub Actions](#configuring-the-github-actions)
     - [Triggering the Rebuild](#triggering-the-rebuild)
-  - [Package Versioning](#package-versioning)
+    - [Devel Package Versioning](#devel-package-versioning)
```

Comment:
Oh, this `obs_integration.md` file is a leftover. Please, update the [obs_integration.md from the website repository instead](https://github.com/agama-project/agama-project.github.io/blob/main/docs/devel/obs_integration.md).